tableTheory/tableTheory.tex

```diff
@@ -213,23 +224,38 @@ \subsubsection{Tuple Selection}
   )
 )
 \end{verbatim}
-where $1 \leq \syntax{n} \leq \syntax{j}$.
-The term $\syntax{((\_\ sel\ n)\ t)}$ is
+When $1 \leq n \leq j$,
```

Comment:
```suggestion
When $0 \leq n \leq j-1$,
```
In cvc5, indices for tuple.select are zero-based.

Reply:
Good catch